src/util.ts

```diff
@@ -262,3 +274,36 @@ export async function writeFileResults(
     }
   }));
 }
+
+/**
+ * A helper function for working with Node's core execFile() method.
+ */
+export async function exec(
+    cwd: string, command: string, args?: string[], options?: ExecOptions):
+    Promise<ExecResult> {
+  const commandOptions = {...options, cwd: cwd} as ExecOptions;
+  try {
+    const {stdout, stderr} = await execFile(command, args, commandOptions);
+    // Trim unneccesary extra newlines/whitespace from exec/execFile output
+    return {stdout: stdout.trim(), stderr: stderr.trim()};
+  } catch (err) {
+    // If an error happens, attach the working directory to the error object
+    err.cwd = cwd;
+    throw err;
+  }
+}
+
+/**
+ * A helper function to check whether a shell command exists or not.
+ * Useful for user-facing warnings.
+ */
+export async function checkCommand(commandName: string): Promise<boolean> {
```

Comment:
no need for this (I don't see it used anyway). It will already have been internally inside of polymer-workspaces, and even still the error thrown by exec is fine for our use-case.

Reply:
Good point. Originally transferred when I was referencing your file to prevent more potential conflicts.

Reply:
Ha, I guess I made the same mistake 😛